static/js/components/nl_search_bar/auto_complete_input.tsx

```diff
@@ -79,23 +79,41 @@ export function AutoCompleteInput(
   const [inputActive, setInputActive] = useState(false);
   const [lastAutoCompleteSelection, setLastAutoCompleteSelection] =
     useState("");
+  // Used to reduce sensitivity to scrolling for autocomplete result dismissal.
```

Comment:
can you add what each of these states are actually keeping track of

Reply:
done.